functions/webhooks/line/LineToFlex.ts

```diff
@@ -168,8 +168,15 @@ export const handler = async (
 
     resolve(success(responses.join()));
   } catch (err: any) {
+    /*
+      For now, we are hard coding the error handler as we will need to update
+      the error handler in Techmatters helper library to accomodate the channelType
+    */
+    const error500 = (error: Error, channelType: string) =>
```

Comment:
I think this should be named something like `channelError500` to prevent confusion with the original function, and movved out to the top level

Reply:
I know this is only a POC but eh, will probably end up being used so nice to tidy up any easy stuff